src/Lean/Elab/BuiltinCommand.lean

```diff
@@ -126,6 +182,30 @@ private partial def elabChoiceAux (cmds : Array Syntax) (i : Nat) : CommandElabM
   | Except.ok env   => setEnv env
   | Except.error ex => throwError (ex.toMessageData (← getOptions))
 
+/-- Adds names from other namespaces to the current namespace.
+
+The command `export Some.Namespace (name₁ name₂)` makes `name₁` and `name₂`
+
+- visible in the current namespace without prefix `Some.Namespace`, like `open`, and
+- visible from outside current namespace `N` as `N.name₁` and `N.name₂`.
+
+## Examples
+
+```lean
+namespace Morning.Sky
+  def star := "venus"
+end Morning.Sky
+
+namespace ReExport
+  export Morning.Sky (star)
+  -- `star` is now in scope
+  #check star
+end ReExport
+
+-- `star` is visible in `ReExport`
+#check ReExport.star
```

Comment:
```suggestion
namespace Evening.Sky
  export Morning.Sky (star)
  -- `star` is now in scope
  #check star
end Evening.Sky

-- `star` is visible in `Evening.Sky`
#check Evening.Sky.star
```
I feel like it defeats the point of the "morning star" reference if you don't use "evening star" as the re-export name.

Reply:
fixed